README.md

```diff
@@ -1,2 +1,41 @@
-# Recommendations
-Module that will provide possible recommendations of similar items
+# Project Name
+
+> Project description
+
+This module will handle the rendering and functionality of a recommendations section.
+
+## Related Projects
+
+  - https://github.com/rent2nite/Main-Gallery
+  - https://github.com/rent2nite/Descriptor-and-Reviews
+  - https://github.com/rent2nite/reservations
+  - https://github.com/rent2nite/Recommendations
+
+## Table of Contents
+
+1. [Usage](#Usage)
+1. [Requirements](#requirements)
+1. [Development](#development)
+
+## Usage
+
+> Some usage instructions
+
+## Requirements
+
+An `nvmrc` file is included if using [nvm](https://github.com/creationix/nvm).
+
+- Node 6.13.0
+- etc
+
+## Development
+
+### Installing Dependencies
+
+From within the root directory:
+
+```sh
+npm install -g webpack
+npm install
+```
+
```

Comment:
Readme looks like it is off to a good start. Will need to be revised in the long run, once more functionality is added to the app.